src/main/scala/circt/stage/CIRCTStage.scala

```diff
@@ -22,7 +22,8 @@ trait CLI { this: Shell =>
     ChiselGeneratorAnnotation,
     PrintFullStackTraceAnnotation,
     ThrowOnFirstErrorAnnotation,
-    WarningsAsErrorsAnnotation
+    WarningsAsErrorsAnnotation,
+    ChiselStage.scala
```

Comment:
🧐

Reply:
I likely botched the cherry-pick. This one doesn't apply cleanly between here and the chisel5 branch. 😅